### ISSUE TITLE
Gaps in solid infill

### ISSUE THREAD

**Issue by wildwick69** [supermerill/Slic3r#4156]
### What happened?

Don't know if this is an issure but it surely weakens strength of the model as solid infiil has gaps on a specific area
![gap](https://github.com/supermerill/SuperSlicer/assets/127697145/17f3a376-92fb-4c4e-afc6-f1050ba12f91)

### Project file & How to reproduce

could you pls. see layer 119 to 126 ?
[Top_Idler_Left_BZV31_v2-2.zip](https://github.com/supermerill/SuperSlicer/files/14410343/Top_Idler_Left_BZV31_v2-2.zip)

### Version

2.5.59.8

### Operating system

Win 10

### Printer model

Voron Trident

**Comment by supermerill** [comment]
yes, it's a section with sparse infill as there is nothing on top or bottom.

You can increase external_infill_margin to something like 6.
Or the bridged_infill margin, should also work but this one may have some drawbacks.

**Comment by wildwick69** [comment]
Many thanks, another thing learned I never would have thought of
